healthcareai/supervised_model_trainer.py

```diff
@@ -17,8 +17,8 @@ def __init__(self, dataframe, predicted_column, model_type, impute=True, grain_c
             dataframe (pandas.core.frame.DataFrame): The training data in a pandas dataframe
             predicted_column (str): The name of the prediction column 
             model_type (str): the trainer type - 'classification' or 'regression'
-            impute (bool): True to impute data (mean of numeric columns and mode of categorical ones). False to drop rows
-                that contain any null values.
+            impute (bool): True to impute data (mean of numeric columns and mode of categorical ones). False to drop
+            rows that contain any null values.
             grain_column (str): The name of the grain column
             verbose (bool): Set to true for verbose output. Defaults to False.
```

Comment:
This docstring doesn't agree with the current default of True.

Reply:
Done, place the default to True.